targets/TARGET_NUVOTON/TARGET_NUC472/trng_api.c

```diff
@@ -82,21 +87,21 @@ void trng_free(trng_t *obj)
 int trng_get_bytes(trng_t *obj, uint8_t *output, size_t length, size_t *output_length)
```

Comment:
`trng_get_bytes` looks pretty similar between `TARGET_NUC472` and `TARGET_M480`. Would it be prudent to move it to a location shared by both `TARGET_NUC472` and `TARGET_M480`?

Reply:
@Patater We will consider to be shared as while one more platform utilize the same HW IP.